docs/current_docs/cookbook/snippets/copy-dir-runtime/python/main.py

```diff
@@ -0,0 +1,13 @@
+import dagger
+from dagger import function, object_type
+
+
+@object_type
+class MyModule:
+    @function
+    async def copy_file(self, source: dagger.File):
+        """Copy a file to the Dagger module runtime container for custom processing"""
+        await source.export("foo.txt")
+        # your custom logic here
+        # for example, read and print the file in the Dagger Engine container
+        print(open("foo.txt").read())
```

Comment:
The standard way to do this in Python is with [pathlib](https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_text):
```diff
- open("foo.txt").read()
+ pathlib.Path("foo.txt").read_text()
```

However, that's a sync (blocking) operation, so in Dagger you should use [anyio](https://anyio.readthedocs.io/en/stable/fileio.html) for asynchronous I/O operations like disk or network access:
```suggestion
        print(await anyio.Path("foo.txt").read_text())
```

Reply:
Don't forget:
```python
import anyio
```